### DISCUSSION TITLE
Project type in custom fields

### DISCUSSION THREAD

**Discussion by oveaurs** [ankitpokhrel/jira-cli#481] [Ideas] [answered]
I was having problems creating issues with jira-cli and got this error msg:
```Error:
  - customfield_<id>: Could not find valid 'id' or 'key' in project object.
```
So I did an attempt on adding support for it:
https://github.com/ankitpokhrel/jira-cli/compare/main...oveaurs:jira-cli:project

This only supports the project key and not the ID though.
With a build from my branch, it works with `--custom custom-field-project=<project key>`

edit: created a PR https://github.com/ankitpokhrel/jira-cli/pull/482

**Comment by ankitpokhrel** [comment, marked as answer]
This is resolved with #482
